src/test/java/uk/nhs/tis/sync/message/publisher/RabbitmqTcsPmUpdatePublisherTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.nhs.tis.sync.message.publisher;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.verify;
+
+import com.google.common.collect.Lists;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.test.util.ReflectionTestUtils;
+
+@ExtendWith(MockitoExtension.class)
+class RabbitmqTcsPmUpdatePublisherTest {
+
+  @InjectMocks
+  RabbitMqTcsPmUpdatePublisher rabbitMqTcsPmUpdatePublisher;
+
+  @Mock
+  RabbitTemplate rabbitTemplate;
+
+  @Captor
+  ArgumentCaptor<String> exchangeNameCaptor;
+
+  @Captor
+  ArgumentCaptor<String> routingKeyNameCaptor;
+
+  @Captor
+  ArgumentCaptor<List<String>> messageCaptor;
+
+  @Test
+  void shouldPublishToRabbitMq() {
+    String routingKeyName = "routingKeyName";
+    String exchangeName = "exchangeName";
+
+    ReflectionTestUtils.setField(
+        rabbitMqTcsPmUpdatePublisher, "routingKey", routingKeyName
+    );
+    ReflectionTestUtils.setField(
+        rabbitMqTcsPmUpdatePublisher, "exchange", exchangeName
+    );
```

Comment:
Is this formatting done by the IDE?  It makes for extra scrolling :disappointed:

Reply:
Yes, this is done by the IDE formatting. But I can format this bit manually. Thanks for spotting.